src/docs/sphinx/developerGuide/Contributing/GitWorkflow.rst

```diff
@@ -124,40 +128,31 @@ should be created to perform the review and merging process.
 See below for details about :ref:`Submitting_a_Pull_Request`.
 
 
-Release Branches
-----------------
+Release Candidate Branches
+--------------------------
 When ``develop`` has progressed to a point where we would like to create a new 
-``release``, we will create a ``release`` branch with the name consisting 
-of ``major.minor.x`` number, where the ``x`` represents the sequence of patch tags that
+``release``, we will create a release candidate branch with the name consisting 
+of ``release_major.minor.x`` number, where the ``x`` represents the sequence of patch tags that
 will be applied to the branch.
 For instance if we were releasing version ``1.2.0``, we would name the branch
-``release/1.2.x``.
-In contrast to the ``OneFlow`` approach, we do not delete the ``release`` branch.
-Once the release candidate is ready, it is tagged and merged back into ``develop``. 
+``release_1.2.x``.
+Once the release candidate is ready, it is merged back into ``develop`` and tagged. 
+Then the ``develop`` branch is merged into the ``release`` branch.
 From that point the ``release`` branch exists to provide a basis for maintaining 
-the major/minor release via the patching index.
-Once the ``release/major.minor.x`` branch is tagged, it only exists as a starting point 
-for patches. 
-Each patch on the branch is tagged using an increment in the patch number, 
-resulting in a sequence of tags like ``release/1.2.0, release/1.2.1, release/1.2.2``, etc. 
-
-.. note::
 
-   In our git workflow, we do not close the release loop by merging the final 
-   release commit into ``develop``. 
-   Once the initial ``release/x.y.z`` branch is tagged ``release/x.y.0`` and merged 
-   into ``develop``, no further efforts are made to merge subsequent modifications 
-   in the ``release`` branch into ``develop``.
-
-An example lifecycle diagram for a release branch:
+An example lifecycle diagram for a release candidate branch:
 
 .. code-block:: sh
 
-   A----B----C----D----E----F   (develop)
-         \            / \
-          \          /   \
-          BA--------BB    EA     (release/1.2.x)
-                        (1.2.0)
+   A----B----C----D----E----F-----G               (release)
+        ^                         ^
+        |                         |
+        |                         |
+   A----B----C----D----E----F-----G------------   (develop)
+         \            / \        / (tag v1.2.0)
```

Comment:
The tags go on develop? I'm not sure it matters but I think it's easiest with a GitHub release to tag the actual release branch (release in this case).

Reply:
Tags are related to commits, but we should tag commits from release branch even if they exist on develop.